README.md

```diff
@@ -18,15 +18,14 @@ This set of instructions will create your own copy of this boilerplate directory
 3. Set up the two following variables and execute the `bootstrap.sh` script to initialise the git repository and replace all references to `openfisca_country_template` with references to your new country package in the code base:
 
 ```sh
-export COUNTRY_NAME=France  # set the name of your country here; you should keep all capitals, and replace any spaces in the name by underscores
-export REPOSITORY_URL=https://github.com/YOUR_ORGANISATION/OpenFisca-$COUNTRY_NAME  # set here the URL of the repository you created in step 2.
 ./bootstrap.sh
 ```
+(Note: you may need to first run `chmod +x ./bootstrap.sh` to grant execute permission)
```

Comment:
Since this is downloaded and not forked, I actually believe this step is mandatory. Either include it in the instructions above, or maybe simply:
````suggestion
bash ./bootstrap.sh
```
````

Reply:
I've structured this differently in a later commit it takes the form:

> 3. Execute the `bootstrap.sh` script to initialise the git repository. This performs numerous tasks including replacing all references to `openfisca-country_template` with references to the new country package. 
>    - Note: if required first run `chmod +x ./bootstrap.sh` to grant execute permission
>    - To execute the script run `./bootstrap.sh` from the command line
>    - After the `bootstrap.sh` has run both it and these instructions are removed.